### PR TITLE
change readme for the couchdb project

### PR DESCRIPTION
**Pull request by robertkowalski**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,33 +1,26 @@
 ## mem3
 
-Mem3 is the node membership application for clustered [CouchDB][1].  It is used in [BigCouch][2] and tracks two very important things for the cluster:
+Mem3 is the node membership application for clustered [CouchDB][1]. It is used in CouchDB version >= 2.0 and tracks two very important things for the cluster:
```
Comment (kxepal) [inline README.md]: May be a little bit human friendly?
"It is used in CouchDB since 2.0 version"

File: README.md [hunk 2]
```diff
@@ -1,33 +1,26 @@
...
 
-A very important point to make here is that BigCouch does not necessarily divide up each database into equal shards across the nodes of a cluster.  For instance, in a 20-node cluster, you may have the need to create a small database with very few documents.  For efficiency reasons, you may create your database with Q=4 and keep the default of N=3.  This means you only have 12 shards total, so 8 nodes will hold none of the data for this database.  Given this feature, we even shard use out across the cluster by altering the 'start' node for the database's shards.
+A very important point to make here is that CouchDB >= 2.0 does not necessarily divide up each database into equal shards across the nodes of a cluster.  For instance, in a 20-node cluster, you may have the need to create a small database with very few documents.  For efficiency reasons, you may create your database with Q=4 and keep the default of N=3.  This means you only have 12 shards total, so 8 nodes will hold none of the data for this database.  Given this feature, we even shard use out across the cluster by altering the 'start' node for the database's shards.
```
Comment (kxepal) [inline README.md]: How about split long text by right 80 chars margin?

File: README.md [hunk 3]
```diff
@@ -1,33 +1,26 @@
...
 ### License
 [Apache 2.0][3]
 
-### Contact
- * [http://cloudant.com][4]
- * [info@cloudant.com][5]
-
 [1]: http://couchdb.apache.org
-[2]: http://github.com/cloudant/bigcouch
+[2]: http://github.com/basho/rebar
```
Comment (kxepal) [inline README.md]: basho/rebar is not a official rebar repo. Use rebar/rebar instead.

**Comment by robertkowalski** [comment]
done :)

File: README.md [hunk 4]
```diff
@@ -1,33 +1,42 @@
...
-
-Splitting and merging shards is an immature feature of the system, and will require attention in the near-term.  We believe we can implement both functions and perform them while the database remains online.
+Both the nodes and shards are tracked in node-local couch databases.  Shards
+are heavily used, so an ETS cache is also maintained for low-latency lookups.
+The nodes and shards are synchronized via continuous CouchDB replication,
+which serves as 'gossip' in Dynamo parlance.  The shards ETS cache is kept in
+sync based on membership and database event listeners.
+
+A very important point to make here is that CouchDB >= 2.0 does not
```
Comment (kxepal) [inline README.md]: >= (;
However, here no need "since" imho an even version mention. Just CouchDB.

File: README.md [hunk 5]
```diff
@@ -1,33 +1,42 @@
...
- * [http://cloudant.com][4]
- * [info@cloudant.com][5]
-
-[1]: http://couchdb.apache.org
-[2]: http://github.com/cloudant/bigcouch
-[3]: http://www.apache.org/licenses/LICENSE-2.0.html
-[4]: http://cloudant.com
-[5]: mailto:info@cloudant.com
-[6]: http://github.com/basho/rebar
+[1]: http://couchdb.apache.org [2]: http://github.com/rebar/rebar [3]:
```
Comment (kxepal) [inline README.md]: Oh no! Someone had stole newlines here!

**Comment by robertkowalski** [comment]
ok second try :)

**Comment by kxepal** [comment]
LGFM

**Comment by robertkowalski** [comment]
merged as fb5f56c21a9d855c1eff2e1fdadadd940213bffe and f62d43836972b2317784c397e69082a0850d74f4

**Pull request merged**